tools/log_processing/broker_registry_counts.rb

```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/env ruby
 RAILS_ROOT = File.expand_path(File.join(__dir__, %w(.. ..)))
-require 'manageiq-gems-pending'
```

Comment:
Should this have been a change as opposed to a removal ?
```suggestion
require 'manageiq/gems/pending'
```

Reply:
Thanks!